<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.3UnAh_c' shared service.

return $this->privates['.service_locator.3UnAh_c'] = new \Symfony\Component\DependencyInjection\ServiceLocator(['fid' => function (): \App\Entity\Friends {
    return ($this->privates['.errored..service_locator.3UnAh_c.App\\Entity\\Friends'] ?? $this->load('getFriendsService.php'));
}, 'pid' => function (): \App\Entity\User {
    return ($this->privates['.errored..service_locator.3UnAh_c.App\\Entity\\User'] ?? $this->load('getUserService.php'));
}, 'uid' => function (): \App\Entity\User {
    return ($this->privates['.errored..service_locator.3UnAh_c.App\\Entity\\User'] ?? $this->load('getUserService.php'));
}]);
